scripts/antlr4/Cypher.g4

```diff
@@ -393,8 +394,16 @@ kU_PropertyDefinitions : kU_PropertyDefinition ( SP? ',' SP? kU_PropertyDefiniti
 
 kU_PropertyDefinition : kU_ColumnDefinition ( SP kU_Default )? ;
 
+kU_PropertyDefinitionBeforeConstraint : kU_ColumnDefinition ( SP kU_Default )? ;
```

Comment:
Don't add these new rules. You should modify kU_PropertyDefinition to make it also accept primary key. Something like

```
kU_PropertyDefinition : kU_ColumnDefinition ( SP kU_Default )?  ( SP PRIMARY SP KEY)?;
```

Reply:
I believe this will cause problems, where a faulty statement with multiple `PRIMARY KEY` specified will be accepted. In my understanding:
`kU_PropertyDefinition` doesn't know whether the other `kU_PropertyDefinition` within the same `kU_PropertyDefinitions` has `PRIMARY KEY` defined. If we use the original rules and only change `kU_PropertyDefinition`, cases where no `PRIMARY KEY` or multiple `PRIMARY KEY` will pass while they are suppose to be flagged.